lib/controllers/v1/taxa_controller.js

```diff
@@ -1142,10 +1142,16 @@ TaxaController.suggest = async req => {
     r.taxon = taxa.find( t => t.id === r.taxon.id );
     return r;
   } );
-  response.results = _.sortBy( response.results, r => (
-    req.suggestParams.order_by === "taxonomy"
-      ? `${r.taxon.ancestry}/${r.taxon.id}` : ( r.score * -1 )
-  ) );
+  response.results = _.sortBy( response.results, r => {
+    switch ( req.suggestParams.order_by ) {
+      case "taxonomy":
+        return `${r.taxon.ancestry}/${r.taxon.id}`;
+      case "scientific_name":
```

Comment:
Can you change this to `sciname` please? Shorter, still confers the meaning.

Reply:
Hey @kueda, so my thought here with https://github.com/inaturalist/inaturalist/pull/3115 and the commit changing it to `scientific_name` was to keep the keys both descriptive and consistent with what is already in place, however I see `ChooserPopover` is setup to accept an I18n label override map so this would be easy enough to do.

Reply:
Ah, hadn't thought about that kind of consistency. Ok, we can roll with what you've got here. Thanks again